src/test/java/pl/put/poznan/transformer/rest/FindScenarioStepNamesNotStartingWithActorServiceControllerTest.java

```diff
@@ -0,0 +1,51 @@
+package pl.put.poznan.transformer.rest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import pl.put.poznan.transformer.logic.domain.dto.ScenarioDTO;
+import pl.put.poznan.transformer.logic.service.FindScenarioStepNamesNotStartingWithActorService;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FindScenarioStepNamesNotStartingWithActorServiceControllerTest {
+    private FindScenarioStepNamesNotStartingWithActorServiceController findScenarioStepNamesNotStartingWithActorServiceController;
+
+    @Mock
+    private FindScenarioStepNamesNotStartingWithActorService findScenarioStepNamesNotStartingWithActorService;
```

Comment:
please make this name shorter, just the variable, call it 'service' if the class name is that long